beginner_source/blitz/neural_networks_tutorial.py

```diff
@@ -44,19 +44,19 @@ class Net(nn.Module):
 
     def __init__(self):
         super(Net, self).__init__()
-        # 1 input image channel, 6 output channels, 3x3 square convolution
-        # kernel
+        # 입력 이미지 채널 1개, 출력 채널 6개, 3x3의 정사각 컨볼루션 행렬
+        # 컨볼루션 커널 정의
         self.conv1 = nn.Conv2d(1, 6, 3)
         self.conv2 = nn.Conv2d(6, 16, 3)
-        # an affine operation: y = Wx + b
-        self.fc1 = nn.Linear(16 * 6 * 6, 120)  # 6*6 from image dimension
+        # 아핀 연산: y = Wx + b
+        self.fc1 = nn.Linear(16 * 6 * 6, 120)  # 6*6은 이미지 차원에 해당
         self.fc2 = nn.Linear(120, 84)
         self.fc3 = nn.Linear(84, 10)
 
     def forward(self, x):
-        # Max pooling over a (2, 2) window
+        # (2, 2) 크기 윈도우에 대해 맥스 풀링(Max pooling)
```

Comment:
```suggestion
        # (2, 2) 크기 윈도우에 대해 맥스 풀링(max pooling)
```

고유 명사가 아니므로 영어 문장의 대소문자 법을 따르면 좋을 것 같습니다 🦕

Reply:
오 좋습니다! 반영하겠습니다 :>